jetstream/bigquery_client.py

```diff
@@ -56,6 +57,14 @@ def _current_timestamp_label(self) -> str:
         """Returns the current UTC timestamp as a valid BigQuery label."""
         return str(int(time.time()))
 
+    def check_if_table_exists(self, table_name: str) -> bool:
+        table_ref = self.client.dataset(self.dataset).table(table_name)
+        try:
+            self.client.get_table(table_ref)
+            return True
+        except NotFound:
+            return False
+
```

Comment:
I'm pretty sure these are practically equivalent and that this is probably my non-python background coming through, but just to ensure that there's always a return (or an unhandled exception):
```suggestion
        try:
            self.client.get_table(table_ref)
        except NotFound:
            return False
        
        return True

```